PluralKit.Bot/Utils/ContextUtils.cs

```diff
@@ -161,7 +161,7 @@ async Task<int> PromptPageNumber()
                         }
                         catch (TimeoutException)
                         {
-                            await ctx.Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?");
+                            await ctx.Reply($"{Emojis.Error} Operation timed out, sorry. Try again, perhaps?", null, null, true);
```

Comment:
i'd use `referenceCommand: true` just to avoid the `, null, null, ` - makes it clear which parameter is being set